src/scout_apm/api/__init__.py

```diff
@@ -44,3 +44,50 @@ def tag(self, key, value):
         if self.span is not None:
             self.span.tag(key, value)
 
+
+class Transaction(ContextDecorator):
+    def __init__(self, name, tags={}):
+        self.name = name
```

Comment:
It's a bit awkward to use `name` and `operation` interchangeably. I'd say we decide and use one or the other.

Reply:
Also should we have a setter method to change the name/operation after the start, in cases where we don't know the name until further into the request? I don't know the guidelines on when to use setter method vs setting attribute directly.

Reply:
In a `Controller/Foo` - the whole thing is operation, the `Foo` is name.

Reply:
Then we should change the `operation` argument name in `start()` to `name` to be consistent, I think.